ios/HeapSettings.bundle/generate_settings_common.rb

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby
+
+require 'fileutils'
+
+def perform_substitution(dir, app_id_dev, app_id_prod)
+  template = File.read File.join(dir, 'HeapSettings.plist.template')
+  output_path = File.join dir, 'HeapSettings.plist'
+
+  settings = template
+    .gsub('__HEAP_APP_ID_DEV__', app_id_dev || '')
+    .gsub('__HEAP_APP_ID_PROD__', app_id_prod || '')
+
+  IO.write output_path, settings
+end
+
+def get_heap_settings_from_json(json)
```

Comment:
Well, damn... this is just beautiful now...

Reply:
🚫code duplication